src/lib/discord/commando/index.js

```diff
@@ -60,6 +62,7 @@ class DiscordCommando {
 			this.client.logs = this.logs
 			this.client.dts = this.dts
 			this.client.re = this.re
+			this.client.updatedDiff = this.updatedDiff()
```

Comment:
doesn't work tho. 

Any idea why this or down by L71 doesn't expose it in client.updatedDIff the command itself?

Reply:
'client' inside the commands actually a poracleDiscordState or poracleTelegramState which provides a generalised interface, probably needs to be carried through to there - on mobile now so can't look

Reply:
got that import resolved. 

telegram didn't use objectDiff as far as I saw

Reply:
The commands are the same for telegram and discord.  Discord capability is encapsulated in poracleDiscordState, poracleDiscordMessage and poracleDiscordUtil; analogous classes exist for telegram to provide compatible capability so that by the time commands are running they don't care about the communication mechanism.
So poracleTelegramState will need similar massaging

Reply:
right... so 
https://github.com/KartulUdus/PoracleJS
already carries over

Reply:
Right - so poracleTelegramState (which is created in there) needs updatedDiff from somewhere